x-pack/plugins/uptime/public/components/queries/ping_list/ping_list.tsx

```diff
@@ -156,7 +156,7 @@ export class Pings extends React.Component<PingListProps, PingListState> {
                 ),
             },
           ];
-          const hasStatus = allPings.reduce(
+          const hasStatus = allPings.pings.reduce(
```

Comment:
We should've destructured this from the `data` prop, but it's not worth adding another commit, we can do it in a later PR.

Reply:
What does that actually buy us vs. dot notation? Is it purely for style, or is there a functional reason to do it?

Reply:
No functional reason, just what the style guide recommends.